pyproject.toml

```diff
@@ -34,6 +34,7 @@ include = ["LICENSE", "src/viam/rpc/libviam_rust_utils.*"]
   nbmake = "^1.4.5"
   types-pillow = "^10.0.0.3"
   mypy-protobuf = "^3.5.0"
+  astroid = "<=2.15.6"
```

Comment:
Can we add a `TODO` or create a ticket for like, 4 weeks from now to readdress this?